sonic-pcied/tests/test_DaemonPcied.py

```diff
@@ -0,0 +1,226 @@
+        daemon_pcied.signal_handler(pcied.signal.SIGUSR1, None)
+        assert daemon_pcied.log_warning.call_count == 1
+        daemon_pcied.log_warning.assert_called_with("Caught unhandled signal 'SIGUSR1' - ignoring...")
+        assert daemon_pcied.log_info.call_count == 0
+        assert daemon_pcied.stop_event.set.call_count == 0
+        assert pcied.exit_code == 0
+
+    @mock.patch('pcied.load_platform_pcieutil', mock.MagicMock())
+    def test_run(self):
+        daemon_pcied = pcied.DaemonPcied(SYSLOG_IDENTIFIER)
+        daemon_pcied.check_pcie_devices = mock.MagicMock()
+
+        daemon_pcied.run()
+        assert daemon_pcied.check_pcie_devices.call_count == 1
+
+    @mock.patch('pcied.load_platform_pcieutil', mock.MagicMock())
+    def test_check_pcie_devices(self):
+        daemon_pcied = pcied.DaemonPcied(SYSLOG_IDENTIFIER)
+        daemon_pcied.update_pcie_devices_status_db = mock.MagicMock()
+        daemon_pcied.check_n_update_pcie_aer_stats = mock.MagicMock()
+        pcied.platform_pcieutil.get_pcie_check = mock.MagicMock()
+
+        daemon_pcied.check_pcie_devices()
+        assert daemon_pcied.update_pcie_devices_status_db.call_count == 1
+        assert daemon_pcied.check_n_update_pcie_aer_stats.call_count == 0
+
+
+    @mock.patch('pcied.load_platform_pcieutil', mock.MagicMock())
+    def test_update_pcie_devices_status_db(self):
+        daemon_pcied = pcied.DaemonPcied(SYSLOG_IDENTIFIER)
+        daemon_pcied.status_table = mock.MagicMock()
+        daemon_pcied.log_info = mock.MagicMock()
+        daemon_pcied.log_error = mock.MagicMock()
+
+        # test for pass resultInfo
+        daemon_pcied.update_pcie_devices_status_db(0)
+        assert daemon_pcied.status_table.set.call_count == 1
+        assert daemon_pcied.log_info.call_count == 1
+        assert daemon_pcied.log_error.call_count == 0
+
+        daemon_pcied.status_table.set.reset_mock()
+        daemon_pcied.log_info.reset_mock()
+
+        # test for resultInfo with 1 device failed to detect
+        daemon_pcied.update_pcie_devices_status_db(1)
+        assert daemon_pcied.status_table.set.call_count == 1
+        assert daemon_pcied.log_info.call_count == 0
+        assert daemon_pcied.log_error.call_count == 1
+
+
+    @mock.patch('pcied.load_platform_pcieutil', mock.MagicMock())
+    @mock.patch('pcied.read_id_file')
+    def test_check_n_update_pcie_aer_stats(self, mock_read):
+        daemon_pcied = pcied.DaemonPcied(SYSLOG_IDENTIFIER)
+        daemon_pcied.device_table = mock.MagicMock()
+        daemon_pcied.update_aer_to_statedb = mock.MagicMock()
+        pcied.platform_pcieutil.get_pcie_aer_stats = mock.MagicMock()
+
+        mock_read.return_value = None
+        daemon_pcied.check_n_update_pcie_aer_stats(0,1,0)
+        assert daemon_pcied.update_aer_to_statedb.call_count == 0
+        assert daemon_pcied.device_table.set.call_count == 0
+        assert platform_pcieutil.get_pcie_aer_stats.call_count == 0
```

Comment:
Test is failing due to "NameError: global name 'platform_pcieutil' is not defined"
I belive you meant to write:
```suggestion
        assert pcied.platform_pcieutil.get_pcie_aer_stats.call_count == 0
```